docs/guides/webview-versions.md

```diff
@@ -0,0 +1,190 @@
+# Webview Versions
+
+## WebView2 (Edge/Chromium)
+
+Tauri uses WebView2 which is based on Microsoft Edge and therefore Chromium. WebView2 can update itself, you are guaranteed a relatively recent chromium build on all Windows targets.
```

Comment:
```suggestion
Tauri uses WebView2 which is what powers Microsoft Edge (and is built on Chromium). Since WebView2 can update itself through OS updates, you are guaranteed a relatively recent Chromium build on all Windows targets.
```

Reply:
> what powers

Nope. they are separate, though webview2 is basicallyyy a modified edge, or _used to be_. It's a little tricky to defined now that the can hardlink each other

> through OS updates

idk, that almost sounds to much like the monthly windows updates. webview2 updates separately from that, similar to how browser can update themselves (other than safari lol)